P5/Source/Guidelines/en/SA-LinkingSegmentationAlignment.xml

```diff
@@ -887,9 +887,11 @@ which is either an <rs>XPATH</rs> as defined above or an
 <rs>IDREF</rs>, the value of an <att>xml:id</att>
 occurring in the document addressed by the base URI in effect
 for the pointer.</p>
-<p>Example: the pointer <code>#left(//gap[1])</code> 
+<p>Example: the pointer <code>#left(//supplied)</code>
 indicates the point between the first <code>lb</code> and the first
-  <code>gap</code> in the <ref target="#SATSXP-ex">example</ref> above.</p>
+<code>supplied</code> in the <ref target="#SATSXP-ex">example</ref> above.</p>
+<p>Example: <code>#left(//gap[1])</code> indicates the point immediately before
+the first <code>gap</code> element in line two and the string <code>si</code>.</p>
```

Comment:
This is simply mis-written. (It boils down to “… the point immediately before A and the string B”, i.e. is missing the word “after” or “following”.) I think you mean something like
~~~xml
<p>Example: <code>#left(//gap[1])</code> indicates the point
immediately before the first <code>gap</code> element in line
two, and after the string <code>si</code>, which is
found immediately after the <tag>lb n="2"</tag>.</p>
~~~
That said, it might be clearer if written “after string B and before A”.